src/main/java/org/sonarlint/cli/InputFileFinder.java

```diff
@@ -153,5 +154,15 @@ public Charset getCharset() {
     public <G> G getClientObject() {
       return null;
     }
+
+    @Override
+    public InputStream inputStream() throws IOException {
+      return Files.newInputStream(path);
+    }
+
+    @Override
+    public String contents() throws IOException {
+      return new String(Files.readAllBytes(path));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this use of constructor "String(byte[])" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1943)